src/System.Security.Principal.Windows/tests/WellKnownSidTypeTests.cs

```diff
@@ -0,0 +1,114 @@
+    [InlineData(WellKnownSidType.BuiltinIncomingForestTrustBuildersSid)]
+    [InlineData(WellKnownSidType.BuiltinPerformanceMonitoringUsersSid)]
+    [InlineData(WellKnownSidType.BuiltinPerformanceLoggingUsersSid)]
+    [InlineData(WellKnownSidType.BuiltinAuthorizationAccessSid)]
+    [InlineData(WellKnownSidType.WinBuiltinTerminalServerLicenseServersSid)]
+    [InlineData(WellKnownSidType.WinBuiltinTerminalServerLicenseServersSid)]
+    [InlineData(WellKnownSidType.WinBuiltinDCOMUsersSid)]
+    [InlineData(WellKnownSidType.WinBuiltinIUsersSid)]
+    [InlineData(WellKnownSidType.WinIUserSid)]
+    [InlineData(WellKnownSidType.WinBuiltinCryptoOperatorsSid)]
+    [InlineData(WellKnownSidType.WinUntrustedLabelSid)]
+    [InlineData(WellKnownSidType.WinLowLabelSid)]
+    [InlineData(WellKnownSidType.WinMediumLabelSid)]
+    [InlineData(WellKnownSidType.WinHighLabelSid)]
+    [InlineData(WellKnownSidType.WinSystemLabelSid)]
+    [InlineData(WellKnownSidType.WinWriteRestrictedCodeSid)]
+    [InlineData(WellKnownSidType.WinCreatorOwnerRightsSid)]
+    [InlineData(WellKnownSidType.WinCacheablePrincipalsGroupSid)]
+    [InlineData(WellKnownSidType.WinNonCacheablePrincipalsGroupSid)]
+    [InlineData(WellKnownSidType.WinEnterpriseReadonlyControllersSid)]
+    [InlineData(WellKnownSidType.WinAccountReadonlyControllersSid)]
+    [InlineData(WellKnownSidType.WinBuiltinEventLogReadersGroup)]
+    [InlineData(WellKnownSidType.WinNewEnterpriseReadonlyControllersSid)]
+    [InlineData(WellKnownSidType.WinBuiltinCertSvcDComAccessGroup)]
+    [InlineData(WellKnownSidType.WinMediumPlusLabelSid)]
+    [InlineData(WellKnownSidType.WinLocalLogonSid)]
+    [InlineData(WellKnownSidType.WinConsoleLogonSid)]
+    [InlineData(WellKnownSidType.WinThisOrganizationCertificateSid)]
+    [InlineData(WellKnownSidType.WinApplicationPackageAuthoritySid)]
+    [InlineData(WellKnownSidType.WinBuiltinAnyPackageSid)]
+    [InlineData(WellKnownSidType.WinCapabilityInternetClientSid)]
+    [InlineData(WellKnownSidType.WinCapabilityInternetClientServerSid)]
+    [InlineData(WellKnownSidType.WinCapabilityPrivateNetworkClientServerSid)]
+    [InlineData(WellKnownSidType.WinCapabilityPicturesLibrarySid)]
+    [InlineData(WellKnownSidType.WinCapabilityVideosLibrarySid)]
+    [InlineData(WellKnownSidType.WinCapabilityMusicLibrarySid)]
+    [InlineData(WellKnownSidType.WinCapabilityDocumentsLibrarySid)]
+    [InlineData(WellKnownSidType.WinCapabilitySharedUserCertificatesSid)]
+    [InlineData(WellKnownSidType.WinCapabilityEnterpriseAuthenticationSid)]
+    [InlineData(WellKnownSidType.WinCapabilityRemovableStorageSid)]
+    public void CanCreateSecurityIdentifierFromWellKnownSidType(WellKnownSidType sidType)
+    {
+        var currentDomainSid = WindowsIdentity.GetCurrent().Owner.AccountDomainSid;
+        var wellKnownSidInstance = new SecurityIdentifier(sidType, currentDomainSid);
+
+        Assert.True(wellKnownSidInstance.IsWellKnown(sidType));
+    }
+}
```

Comment:
Can you please add negative test case to test WinCapabilityRemovableStorageSid+1 (and probably also -1)?

Also I would add compat test MaxDefined == WinBuiltinTerminalServerLicenseServersSid. To prevent accidental changes to it in future.